cli/lib/kontena/cli/vault/update_command.rb

```diff
@@ -7,13 +7,18 @@ class UpdateCommand < Kontena::Command
 
     option ['-u', '--upsert'], :flag, 'Create secret unless already exists', default: false
     option '--silent', :flag, "Reduce output verbosity"
+    option ['-i', '--stdin'], :flag, 'Read value from stdin', default: false
```

Comment:
$stdin.tty? is false when some kind of piping is going on so the option is not needed.